sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -724,14 +761,16 @@ def test_uniform_targets():
     for model in models_single_task:
         for y_values in (0, 5):
             y1.fill(y_values)
-            assert_array_equal(model.fit(X_train, y1).predict(X_test), y1)
+            with pytest.warns(ConvergenceWarning):
+                assert_array_equal(model.fit(X_train, y1).predict(X_test), y1)
```

Comment:
It would be best to store the prediction and then make the check. I find the pattern of having an assert in `pytest.warns` a bit weird.

Reply:
Something like
```
            with pytest.warns(ConvergenceWarning):
                pred = model.fit(X_train, y1).predict(X_test)
            assert_array_equal(pred, y1)
            assert_array_equal(model.alphas_, [np.finfo(float).resolution]*3)
```

Reply:
Done that now.